test/fixtures/test-runner/no-isolation/global-hooks.js

```diff
@@ -1,6 +0,0 @@
-import('node:test').then((test) => {
```

Comment:
can we add tests for both require and import in cjs?

Reply:
There is already in this PR, no?

Reply:
I might be missing it but where is there a cjs test with import?

Reply:
OH, sorry I think I misunderstood. You want

```
node --import global.cjs test.js
node --require global.cjs test.js // this exists
```

Reply:
Done (and the import CJS case passes): https://github.com/nodejs/node/pull/57595/commits/6a38d3246b89fe3d101b085187a9f2df662fc65c

Reply:
@JakobJingleheimer if nobody precedes me, I'll take a look ASAP.

Reply:
@pmarchini looks like nobody is, so if you could, that would be great 😀